src/sentry/static/sentry/app/views/performance/realUserMonitoring/content.tsx

```diff
@@ -3,18 +3,29 @@ import {Location} from 'history';
 import {browserHistory} from 'react-router';
 import styled from '@emotion/styled';
 
+import {t} from 'app/locale';
 import CreateAlertButton from 'app/components/createAlertButton';
 import * as Layout from 'app/components/layouts/thirds';
 import {getParams} from 'app/components/organizations/globalSelectionHeader/getParams';
 import space from 'app/styles/space';
-import {Organization, Project} from 'app/types';
+import {Organization, Project, SelectValue} from 'app/types';
 import EventView from 'app/utils/discover/eventView';
 import {decodeScalar} from 'app/utils/queryString';
 import SearchBar from 'app/views/events/searchBar';
+import DropdownControl, {DropdownItem} from 'app/components/dropdownControl';
 
 import TransactionVitals from './transactionVitals';
 import TransactionHeader, {Tab} from '../transactionSummary/header';
 
+const FILTER_OPTIONS: SelectValue<string>[] = [
+  {label: t('Upper outer fence'), value: 'upper_outer_fence'},
+  {label: t('Upper inner fence'), value: 'upper_inner_fence'},
```

Comment:
Would calling these 'Outer fence' and 'Inner fence' work?  Both still feel like terms we're going to have to explain as customers will not likely be educated on them.